### PR TITLE
Simple conditions content

### PR DESCRIPTION
**Pull request by teneightfive**
- [x] Earache
- [x] Nosebleeds
- [x] Hay fever
- [x] Sprains and strains

### CODE REVIEW

File: content/conditions/earache/main-content-2.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+## Your pharmacist can help with earaches
+
+They can give advice and recommend treatments, such as eardrops.
+
+!!! info_compact
+[Find a pharmacy]({{ findersBaseUrl }}finders/find-help)
+!!!
+
+!!! attention
+## See a GP if:
```
Comment (tyom) [inline content/conditions/earache/main-content-2.md]: Colon?

File: content/conditions/hay-fever/main-content-1.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+If you have asthma, you might also:
+
+- have a tight feeling in the chest
+- be short of breath
+- wheeze and cough
+
+Hay fever will last for weeks or months unlike a cold which usually goes away
+after 1 to 2 weeks.
+
+
```
Comment (tyom) [inline content/conditions/hay-fever/main-content-1.md]: Why two new lines?

File: content/conditions/hay-fever/main-content-2.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- a blocked nose
+
+Your pharmacist can advise you on which ones are best for you.
+
+!!! info_compact
+[Find a pharmacy]({{ findersBaseUrl }}finders/find-help)
+!!!
+
+!!! attention
+## See a GP if:
```
Comment (tyom) [inline content/conditions/hay-fever/main-content-2.md]: Colon again.

File: content/conditions/nosebleed/main-content.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+!!! attention
+## See a GP if:
```
Comment (tyom) [inline content/conditions/nosebleed/main-content.md]: 🚨

File: content/conditions/nosebleed/main-content.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+- it feels like your heart isn’t beating normally or you’re breathless
+- you’re taking a blood-thinning medicine, such as warfarin
+- you have a condition that means your blood can’t clot properly, such
+  as haemophilia
+!!!
+
+Your GP might want to test you for haemophilia or for other conditions such as anaemia.
+
+!!! alert
+## See a GP now or go to A&E if:
```
Comment (tyom) [inline content/conditions/nosebleed/main-content.md]: 🚨

File: content/conditions/sprains-and-strains/main-content.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+## How long will it take for a sprain or strain to heal?
+
+- After two weeks: most sprains and strains will feel better.
+- Up to 8 weeks: avoid strenuous exercise such as running, as there’s a risk of
+  further damage.
+
+More severe sprains and strains can take several months to get back to normal.
+
+!!! attention
+## Go to a minor injuries unit or your GP if:
```
Comment (tyom) [inline content/conditions/sprains-and-strains/main-content.md]: 🚨

File: content/conditions/sprains-and-strains/main-content.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+For severe sprains and strains, your GP may refer you to a physiotherapist. You
+may also choose to book appointments privately.
+
+!!! info_compact
+[Find a physiotherapist](http://www.csp.org.uk/your-health/find-physio/physio2u)
+!!!
+
+!!! severe
+## Go to A&E or call 999:
```
Comment (tyom) [inline content/conditions/sprains-and-strains/main-content.md]: 🚨

File: content/conditions/earache/main-content-2.md (hunk 1 above)
Comment (teneightfive) [inline content/conditions/earache/main-content-2.md]: Yep, that's now the correct pattern. There was [a change](https://github.com/nhsuk/betahealth/commit/d880425e91518477455dd45d9552d90a6633754b) whilst you were away to replace them all.

File: assets/stylesheets/units/_lists.scss [hunk 8]
```diff
@@ -20,11 +20,19 @@ ol {
...
+
+  dt {
+    @include bold-font(20);
+  }
 }
 
 .local-header,
 .page-section,
 .global-footer {
+  dd {
```
Comment (tyom) [inline assets/stylesheets/units/_lists.scss]: Could it be more generic: `dt + dd, dd + dd`?

**Pull request merged**